astroquery/esa/hubble/tests/test_esa_hubble_remote.py

```diff
@@ -58,6 +60,7 @@ def test_query_hst_tap_async(self):
         assert "observation_id" in result.keys()
         remove_last_job()
 
+    @pytest.mark.skipif("SKIP_SLOW")
```

Comment:
I wonder whether getting an observation_id that results in a small download is possible, that would definitely my preference as opposed to skipping.

Reply:
Agreed. I'm planning to try to sort out a bunch of the slow tests in a seperate PR. I'll make an issue to capture this.

Reply:
I would say to just remove this change for now, this test is not particularly slow and we may still want to run it as is in the weekly cron. (making it faster would be of course more than welcome in a follow-up)